releasenotes/notes/0.13/fix-3925-c0e40ece078f2ce1.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    A custom pass manager was creating a deadlock when run in on multiple circuits in parallel. 
```

Comment:
The release note for this fix isn't needed, since the bug only ever occurred on master.

Reply:
Good catch, I'll remove this now

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/4057/commits/31da6c99ba275c899eaf6f8483d5edac2b596efb